fuel/server.py

```diff
@@ -0,0 +1,177 @@
+import logging
+from collections import deque
+from multiprocessing import Process
+
+import numpy
+import six
+import zmq
+
+from fuel.utils import buffer_
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level='INFO')
+
+
+def send_arrays(socket, arrays, flags=0, copy=True, track=False, stop=False):
+    """Send a NumPy array using the buffer interface and some metadata.
+
+    Parameters
+    ----------
+    socket : :class:`zmq.Socket`
+        The socket to send data over.
+    arrays : list
+        A list of :class:`numpy.ndarray` to transfer.
+    stop : bool, optional
+        Instead of sending a series of NumPy arrays, send a JSON object
+        with a single `stop` key. The :func:`recv_arrays` will raise
+        ``StopIteration`` when it receives this.
+
+    See Also
+    --------
+    :meth:`zmq.Socket.send` for other arguments
```

Comment:
I'm not sure :) I need to read into ZMQ a bit more to know what the effect of these flags would be, but you're probably right that I can remove them from now.